src/components/Provider.svelte

```diff
@@ -25,37 +17,31 @@
   storeToken.subscribe((val) => (token = val.toString()));
 
   let api: ApiPromise;
-  dotApi.subscribe((storeDotApi) => {
+  dotApi.subscribe(async (storeDotApi) => {
     if (storeConnected && storeDotApi.api) {
       api = storeDotApi.api;
     }
   });
 
-  let signingAddress = ''; // eslint-disable-line no-unused-vars
-  transactionSigningAddress.subscribe(async (val) => {
-    signingAddress = val;
+  user.subscribe(async (val) => {
     if (api) {
-      accountBalances = await getBalances(api, val);
+      accountBalances = await getBalances(api, $user.address);
```

Comment:
Isn't val = user? why use this callback if we're just referencing the store instead on line 28?

Reply:
getBalances takes a `string` for account:
`getBalances(apiPromise: ApiPromise, accountId: string): Promise<AccountBalances>
`

Reply:
What do you suggest?